WooCommerce/Classes/ViewModels/OrderDetailsViewModel/OrderPaymentDetailsViewModel.swift

```diff
@@ -51,17 +51,33 @@ final class OrderPaymentDetailsViewModel {
         return currencyFormatter.formatAmount(order.total, with: order.currency) ?? String()
     }
 
+    var paymentTotal: String {
+        if order.datePaid == nil {
+            return currencyFormatter.formatAmount("0.00", with: order.currency) ?? String()
+        }
+
+        return totalValue
+    }
+
     /// Payment Summary
-    /// - returns: A full sentence summary of how much was paid and using what method.
+    /// - returns: A full sentence summary of when was paid and using what method.
     ///
     var paymentSummary: String? {
         if order.paymentMethodTitle.isEmpty {
             return nil
         }
 
+        if order.datePaid == nil {
+            return NSLocalizedString(
+                "Awaiting payment via \(order.paymentMethodTitle)",
+                comment: "Awaiting payment via (payment method title)"
+            )
+        }
+
+        let datePaid = order.datePaid!.toString(dateStyle: .medium, timeStyle: .none)
```

Comment:
I understand this is safe, as the code would have returned with "Awaiting payment..." if datePaid is nil, but I would prefer avoiding force unwraps (except in the tests). You never know when this line is going to be moved to a new placement, or the previous code that makes it safe suddenly disappears.

Reply:
That makes a lot of sense, I will remember it for the future. 👍